scene/2d/gpu_particles_2d.cpp

```diff
@@ -34,6 +34,13 @@
 
 #ifdef TOOLS_ENABLED
 #include "core/config/engine.h"
+#include "editor/editor_node.h"
+
+void GPUParticles2D::_selection_changed() {
```

Comment:
In CI running `GPUParticles2D.new()._selection_changed()` will crash with this backtrace
```
 scene/2d/gpu_particles_2d.cpp:41:74: runtime error: member call on null pointer of type 'struct EditorNode'
scene/2d/gpu_particles_2d.cpp:41:74: runtime error: member access within null pointer of type 'struct EditorNode'

================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.0.dev.custom_build (311267a9362896e1b3a3be1cdc8caab5b51604e6)
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] ./bin/godot.linuxbsd.tools.64s(+0x31e33290) [0x55e0dfdf5290] (??:0)
[2] /lib/x86_64-linux-gnu/libc.so.6(+0x46210) [0x7fa3e884b210] (??:0)
[3] GPUParticles2D::_selection_changed() (??:0)
[4] void call_with_variant_args_helper<__UnexistingClass>(__UnexistingClass*, void (__UnexistingClass::*)(), Variant const**, Callable::CallError&, IndexSequence<>) (??:0)
[5] void call_with_variant_args_dv<__UnexistingClass>(__UnexistingClass*, void (__UnexistingClass::*)(), Variant const**, int, Callable::CallError&, Vector<Variant> const&) (??:0)
[6] MethodBindT<>::call(Object*, Variant const**, int, Callable::CallError&) (??:0)
[7] Object::call(StringName const&, Variant const**, int, Callable::CallError&) (??:0)
[8] Object::callv(StringName const&, Array const&) (??:0)
[9] void call_with_variant_args_ret_helper<__UnexistingClass, Variant, StringName const&, Array const&, 0ul, 1ul>(__UnexistingClass*, Variant (__UnexistingClass::*)(StringName const&, Array const&), Variant const**, Variant&, Callable::CallError&, IndexSequence<0ul, 1ul>) (??:0)
[10] void call_with_variant_args_ret_dv<__UnexistingClass, Variant, StringName const&, Array const&>(__UnexistingClass*, Variant (__UnexistingClass::*)(StringName const&, Array const&), Variant const**, int, Variant&, Callable::CallError&, Vector<Variant> const&) (??:0)
[11] MethodBindTR<Variant, StringName const&, Array const&>::call(Object*, Variant const**, int, Callable::CallError&) (??:0)
[12] GDScriptFunction::call(GDScriptInstance*, Variant const**, int, Callable::CallError&, GDScriptFunction::CallState*) (??:0)
[13] GDScriptInstance::call(StringName const&, Variant const**, int, Callable::CallError&) (??:0)
[14] Object::call(StringName const&, Variant const**, int, Callable::CallError&) (??:0)
[15] Variant::call(StringName const&, Variant const**, int, Variant&, Callable::CallError&) (??:0)
[16] GDScriptFunction::call(GDScriptInstance*, Variant const**, int, Callable::CallError&, GDScriptFunction::CallState*) (??:0)
[17] GDScriptInstance::call(StringName const&, Variant const**, int, Callable::CallError&) (??:0)
[18] Node::_gdvirtual__ready_call() (??:0)
[19] Node::_notification(int) (??:0)
[20] Node::_notificationv(int, bool) (??:0)
[21] Object::notification(int, bool) (??:0)
[22] Node::_propagate_ready() (??:0)
[23] Node::_set_tree(SceneTree*) (??:0)
[24] Node::_add_child_nocheck(Node*, StringName const&) (??:0)
[25] Node::add_child(Node*, bool, Node::InternalMode) (??:0)
[26] void call_with_variant_args_helper<__UnexistingClass, Node*, bool, Node::InternalMode, 0ul, 1ul, 2ul>(__UnexistingClass*, void (__UnexistingClass::*)(Node*, bool, Node::InternalMode), Variant const**, Callable::CallError&, IndexSequence<0ul, 1ul, 2ul>) (??:0)
[27] void call_with_variant_args_dv<__UnexistingClass, Node*, bool, Node::InternalMode>(__UnexistingClass*, void (__UnexistingClass::*)(Node*, bool, Node::InternalMode), Variant const**, int, Callable::CallError&, Vector<Variant> const&) (??:0)
[28] MethodBindT<Node*, bool, Node::InternalMode>::call(Object*, Variant const**, int, Callable::CallError&) (??:0)
[29] GDScriptFunction::call(GDScriptInstance*, Variant const**, int, Callable::CallError&, GDScriptFunction::CallState*) (??:0)
[30] GDScriptInstance::call(StringName const&, Variant const**, int, Callable::CallError&) (??:0)
[31] Node::_gdvirtual__process_call(double) (??:0)
[32] Node::_notification(int) (??:0)
[33] Node::_notificationv(int, bool) (??:0)
[34] CanvasItem::_notificationv(int, bool) (??:0)
[35] Control::_notificationv(int, bool) (??:0)
[36] Object::notification(int, bool) (??:0)
[37] SceneTree::_notify_group_pause(StringName const&, int) (??:0)
[38] SceneTree::process(double) (??:0)
[39] Main::iteration() (??:0)
[40] OS_LinuxBSD::run() (??:0)
[41] ./bin/godot.linuxbsd.tools.64s(main+0x414) [0x55e0dfdf2b9d] (??:0)
[42] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7fa3e882c0b3] (??:0)
[43] ./bin/godot.linuxbsd.tools.64s(_start+0x2e) [0x55e0dfdf26ce] (??:0)
-- END OF BACKTRACE --
================================================================
Aborted (core dumped)
FATAL ERROR: Godot has been crashed.
```

Reply:
The dependency was removed entirely in the latest commit.